app/src/main/res/values/strings.xml

```diff
@@ -236,6 +236,8 @@
     <string name="tx_list_now">Now</string>
     <string name="tx_list_loading_gif">Loading GIF</string>
     <string name="tx_list_retry_loading_gif">Retry loading GIF</string>
+    <string name="tx_list_someone">Someone</string>
+    <string name="tx_list_you_received_one_side_payment">You received a one side payment</string>
```

Comment:
You should also use ➡️ as avatar placeholder and "You recovered some funds" for a custom message used by `wallet_start_recovery`.